.github/workflows/win.yml

```
@@ -75,6 +75,19 @@ jobs:
         run: |
           cmake . -DBUILD_SHARED_LIBS=OFF -DPCRE_SUPPORT_UNICODE_PROPERTIES=ON -DPCRE_SUPPORT_JIT=ON -DPCRE_STATIC_RUNTIME=ON -DCMAKE_FIND_USE_SYSTEM_ENVIRONMENT_PATH=OFF
           cmake --build . --config Release
+      - name: Download libiconv
+        if: steps.cache-libs.outputs.cache-hit != 'true'
+        uses: actions/checkout@v2
+        with:
+          repository: pffang/libiconv-for-Windows
+          ref: 9b7aba8da6e125ef33912fa4412779279f204003 # master @ 2021-08-24
+          path: libiconv
+      - name: Build libiconv
+        if: steps.cache-libs.outputs.cache-hit != 'true'
+        working-directory: ./libiconv
+        run: |
+          [IO.File]::WriteAllLines("$pwd\libiconv.vcxproj", (Get-Content "libiconv.vcxproj").Replace("MultiThreadedDLL", "MultiThreaded"))
```

Can't this be `sed`? We have one instance already below.

```suggestion
          sed -i 's/MultiThreadedDLL/MultiThreaded/' libiconv.vcxproj
```

---

I always wondered why that worked on Windows CI and it seems `sed.exe` comes from the Git Bash.